external_libs/wifi_config_haa/content/index.html

```diff
@@ -42,42 +52,57 @@
 .networks li.other {border-top: 2px solid #aaa; text-align: center;}
 .nonetworks {text-align: center; margin: 1em 0;}
 .field {margin-bottom: 0.5em;}
-.field.required label::before {content: "*"; color: red;}
+label.required::before {content: "*"; color: red;}
 </style></head>
 <body><div class="container">
-    <h1>Home Accessory Architect v0.3.0</h1>
+<h1>Home Accessory Architect v0.3.0</h1>
 <form action="/settings" method="get">
-<input type="submit" id="refresh" value="⟲ Refresh" />
+    <input type="submit" id="refresh" value="⟲ Refresh Page" />
 </form>
+<h3>WIFI Networks</h3>
 <form action="/settings" method="post">
-<div class="field required conf">
-<label for="conf">JSON:</label>
-<textarea id="conf" name="conf">
-<!-- part HTML_SETTINGS_MIDDLE --></textarea>
-<br>
-<input type="checkbox" name="reset" value="y"> Reset HomeKit ID<br>
-<!-- part HTML_SETTINGS_OTA --><input type="checkbox" name="ota" value="y"> OTA Update<br>
-<input type="checkbox" name="autoota" value="y" <!-- part HTML_SETTINGS_AUTOOTA -->> Auto OTA Updates<br>
-<!-- part HTML_SETTINGS_POSTOTA --></div>
-<input type="submit" id="join" value="Save" />
 <div class="nonetworks" style="display: none;">
 No networks found
 </div>
 <ul id="networks" class="networks">
-<!-- part HTML_NETWORK_ITEM -->{% for network in networks %}
-<li class="{{ network.secure | ternary('secure', 'unsecure') }}">{{ network.ssid }}</li>
+<!-- part HTML_NETWORK_ITEM -->
+{% for network in networks %}
+<li class="{{ network.secure | ternary('secure', 'unsecure') }}"> {{ network.ssid }}</li>
 {% endfor %}
-<!-- part HTML_SETTINGS_FOOTER --><li class="other">Enter SSID manually</li>
+<!-- part HTML_SETTINGS_FOOTER -->
+<li class="other">Enter SSID manually</li>
 </ul>
 <div class="field required ssid" style="display: none;">
-<label for="ssid">SSID:</label>
+<label class="required" for="ssid">SSID</label>
 <input type="text" id="ssid" name="ssid" />
 </div>
 <div class="field required password" style="display: none;">
-<label for="password">Password:</label>
+<label class="required" for="password">Password</label>
 <input type="password" id="password" name="password" />
 </div>
+ 
+<h3>HAA Settings</h3>
+<div class="conf">
+<!-- part HTML_SETTINGS_MIDDLE -->
+<label class="required" for="conf">JSON</label>
+<textarea id="conf" name="conf" required >
+</textarea>
+<br>
+<label>
+<input type="checkbox" name="reset" value="y"> Reset HomeKit ID
+</label>
+<!-- part HTML_SETTINGS_OTA -->
+<label>
+<input type="checkbox" name="ota" value="y"> OTA Update
+</label>
+<!-- part HTML_SETTINGS_AUTOOTA -->
+<label>
+<input type="checkbox" name="autoota" value="y"> Auto OTA Updates
```

Comment:
This is broken. Here must be <!-- part HTML_SETTINGS_AUTOOTA --> in the middle. See original code.

Reply:
fixed

Reply:
Mmm, no. Must be:

`<input type="checkbox" name="autoota" value="y" <!-- part HTML_SETTINGS_AUTOOTA -->> Auto OTA Updates<br>`

But don't worry. I'll merge and the I will fix it.

Reply:
It should be like this on the latest commit. This comment references the original commit.

Reply:
Sorry. I don't know where I saw it.